pyccel/ast/numpyext.py

```diff
@@ -191,31 +191,27 @@ def rank(self):
     def fprint(self, printer, lhs):
         """Fortran print."""
 
+        lhs_code = printer(lhs)
```

Comment:
Yes, I did not change the logic of our Numpy printer. I only removed the allocations.

Reply:
Ok. I think it would be nice to move these to the printer at some point so we use the same logic throughout. What do you think? Should we create an issue for this? I think it should be do-able now as the `lhs` is not necessary to create the object anymore

Reply:
Yes, we should create an issue.

Reply:
I've just created issue #531 for this.